src/index.ts

```diff
@@ -41,7 +41,7 @@ const config: Linter.Config = {
   plugins: ['@typescript-eslint'],
   overrides: [
     {
-      files: ['*.ts', '*.tsx'],
+      files: ['*.ts', '*.tsx', '*.vue'],
```

Comment:
How does this work? What manages to parse this?

Reply:
Indeed, good question! My mistake, this PR doesn't work, it can't parse `.vue` file (I just tested it in a newly created `vue` project).

Do you have any ideas, how could we support `.vue` files with ease for `eslint-config-standard-with-typescript`?
Maybe add some documentation to add the support for it?